addons/event/data/res_partner_demo.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0"?>
+<odoo>
+    <data>
+
+    <!-- Res Partner that will be used as locations for events -->
+    <record id="res_partner_location_0" model="res.partner">
```

Comment:
Use already existing partners (maybe tweak one of them if necessary). We have enough demo, no need to pile them up.